sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -23,5 +23,7 @@
     <meta-data android:name="io.sentry.dsn" android:value="https://key@sentry.io/123" />
     <meta-data android:name="io.sentry.debug" android:value="true" />
   </application>
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
why do we need that? `INTERNET` should be enough

Reply:
Why don't you move required permissions to the library project so they are inherited automatically by consuming apps via manifest merging?

Reply:
or that, but not sure if this is a good idea, if I am not wrong, the current SDK crashes if there's no `INTERNET` permission, so we would need to write down the behavior change.

Reply:
Is that because you do not want to add permissions on behalf of the app?

Reply:
@ninniuz yes, only `INTERNET` I don't see as a problem at all, but I'm pretty sure we are gonna require more, then it could be a problem.

Reply:
Once we merge #32 we can delete these two lines.

Reply:
merged

Reply:
tks